src/DeviceInterface/InterfacePrototype.js

```diff
@@ -294,14 +319,28 @@ class InterfacePrototype {
      * @param {string} type
      */
     async selectedDetail (data, type) {
-        this.activeFormSelectedDetail(data, type)
-    }
-
-    /**
-     * @param {IdentityObject|CreditCardObject|CredentialsObject|{email:string, id: string}} data
-     * @param {string} type
-     */
-    activeFormSelectedDetail (data, type) {
+        if (this.globalConfig.isTopFrame) {
+            /**
+             * This is overridden in the Overlay, so that instead of trying to fill a form
+             * with the selected credentials, we instead send a message to the native
+             * side. Once received, the native side will store that selection so that a
+             * subsequence call from main webpage can retrieve it via polling.
+             *
+             * @override
+             * @param detailIn
+             * @param configType
+             * @returns {Promise<void>}
+             */
+            let detailsEntries = Object.entries(data).map(([key, value]) => {
+                return [key, String(value)]
+            })
+            const entries = Object.fromEntries(detailsEntries)
+            // todo: Make this part conform to the new getAutofillData format, so that filling can occur in a uniform manner.
+            // This hasn't been done yet here since it will involve changes to to macOS
```

Comment:
Do you mean `getAutofillInitData`? How is this new format different?

Reply:
This was put here by mistake, a last-min refactor. I'm pushing an update that removes the isTopFrame check